tubular/edx_api.py

```diff
@@ -88,17 +113,19 @@ def _giveup_on_unexpected_exception(exc):
     """
     Giveup method that gives up backoff upon any unexpected exception.
     """
-    return not ( (500 <= exc.response.status_code < 600 and exc.response.status_code != 504) or exc.response.status_code == 104)
+    return not ((
+                        500 <= exc.response.status_code < 600 and exc.response.status_code != 504) or exc.response.status_code == 104)
 
 
 def _retry_lms_api():
     """
     Decorator which enables retries with sane backoff defaults for LMS APIs.
     """
+
     def inner(func):  # pylint: disable=missing-docstring
```

Comment:
@func.wrap() would be a great addition for the internal decorator function

Reply:
it's a parameterized decorator that takes no parameters and I don't see the point in using `wraps`
maybe something like this
```
decorator = func_with_backoff(func_with_timeout_backoff(func))
        decorator = wraps(func)(decorator)
        return decorator
```

Reply:
Oh! I've missed that retry function is based on backoff lib, its functions are already covered by `wraps`